recipes/flac/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+./autogen.sh
+./configure --prefix=${PREFIX} --enable-sse
```

Comment:
For the other xiph libs there were, so probably there are tests. I will add `make check`